lib/Fenix/Model/IRC.pm

```diff
@@ -181,6 +223,91 @@ sub start($self) {
 
 1;
 
+__DATA__
+
+@@ irclog.sql
+
+-- 1 up
+
+CREATE TABLE channel (
+     channel_id INTEGER PRIMARY KEY AUTOINCREMENT,
+     channel TEXT UNIQUE NOT NULL,
+     topic TEXT
+);
+
+CREATE TABLE command (
+     command_id INTEGER PRIMARY KEY AUTOINCREMENT,
+     command TEXT UNIQUE NOT NULL
+);
+
+CREATE TABLE nick (
+     nick_id INTEGER PRIMARY KEY AUTOINCREMENT,
+     nick TEXT UNIQUE NOT NULL
+);
+
+CREATE TABLE message (
+     message_id INTEGER PRIMARY KEY AUTOINCREMENT,
+     ts DATETIME NOT NULL,
+     command_id INTEGER,
+     channel_id INTEGER,
+     nick_id INTEGER,
+     message TEXT,
+     FOREIGN KEY(command_id) REFERENCES command(command_id),
+     FOREIGN KEY(channel_id) REFERENCES channel(channel_id),
+     FOREIGN KEY(nick_id) REFERENCES nick(nick_id)
```

Comment:
```suggestion
     message TEXT
```

Reply:
I've changed this to put the attributes alongside the fields, it looks neater that way, thanks.